sklearn/model_selection/tests/test_validation.py

```diff
@@ -1720,6 +1721,38 @@ def test_fit_and_score_working():
     assert result['parameters'] == fit_and_score_kwargs['parameters']
 
 
+def test_score_failing():
+    iris = load_iris()
+    X, y = iris.data, iris.target
+    clf = MockClassifier()
+    score_kwargs = {'error_score': np.nan}
```

Comment:
I think it's strange that this is only used to format a warning message, and then only one key of it. Remove this variabe.